infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -307,7 +356,7 @@ ChannelFuture startAndListen(@Nonnull EventLoopGroup bossGroup,
             bootstrapConfigurer.configure(bootstrap);
 
             bootstrap.childHandler(getServerChannelInitializer(context, router));
-            return bootstrap.bind(port).sync();
+            return bootstrap.bind(address, port).sync();
```

Comment:
I don't think this is necessary and it seems that it will break the previous behavior which binded to all addresses.

Reply:
We don't want that right? We are not using the `--address` field then.
Shouldn't it bind only to one address.

Reply:
The address field is currently being used only to populate the layout (when -s is specified), nothing else. Is there a use case for listening on only one address?

Reply:
The current implementation would be bug prone if we attempt to run 2 corfu servers on different interfaces with the same port. We can bind to an address if provided or default to binding to all addresses if nothing is specified. Would this be reasonable?

Reply:
I'm concerned about keeping the current behavior.

Does the use case you're concerned about actually exist?
Are there cases where you have a server with two interfaces and want to serve TWO different servers on the same port?